api/db-cdr/generate-cdr/cloudsql-import-bucket.sh

```diff
@@ -0,0 +1,143 @@
+#!/bin/bash
+
+# Imports all sql and csv files in the bucket to the database.
+# IF --create-dump-file is specified, it is ran first
+# Table names for csvs are determined from the filename.
+
+set -xeuo pipefail
+IFS=$'\n\t'
+
+CREATE_DUMP_FILE=
+
+# get options
+USAGE="./generate-clousql-cdr/cloudsql-import-bucket.sh --project <PROJECT> --instance <INSTANCE> --bucket <BUCKET>"
+# example account for test : all-of-us-workbench-test@appspot.gserviceaccount.com
+while [ $# -gt 0 ]; do
+  echo "1 is $1"
+  case "$1" in
+    --project) PROJECT=$2; shift 2;;
+    --instance) INSTANCE=$2; shift 2;;
+    --bucket) BUCKET=$2; shift 2;;
+    --database) DATABASE=$2; shift 2;;
+    --create-dump-file) CREATE_DUMP_FILE=$2; shift 2;;
+    -- ) shift; break ;;
+    * ) break ;;
+  esac
+done
+
+if [ -z "${PROJECT}" ]
+then
+  echo "Usage: $USAGE"
+  exit 1
+fi
+
+if [ -z "${INSTANCE}" ]
+then
+  echo "Usage: $USAGE"
+  exit 1
+fi
+
+if [ -z "${BUCKET}" ]
+then
+  echo "Usage: $USAGE"
+  exit 1
+fi
+
+# Grant Access to files for service account
+SERVICE_ACCOUNT="${PROJECT}@appspot.gserviceaccount.com"
+
+gsutil acl ch -u $SERVICE_ACCOUNT:O gs://$BUCKET/*
+gcloud auth activate-service-account $SERVICE_ACCOUNT --key-file=$GOOGLE_APPLICATION_CREDENTIALS
+
+# Grant access to buckets for service account for cloudsql
+SQL_SERVICE_ACCOUNT=$(gcloud sql instances describe --project $PROJECT \
+    --account $SERVICE_ACCOUNT $INSTANCE | grep serviceAccountEmailAddress \
+    | cut -d: -f2)
+# Trim leading whitespace from sql service account
+SQL_SERVICE_ACCOUNT=${SQL_SERVICE_ACCOUNT//[[:blank:]]/}
+
+echo "Granting GCS access to ${SQL_SERVICE_ACCOUNT} to bucket $BUCKET/*"
```

Comment:
Interesting, I thought the import process did this for you. Or is that just the Cloud Console UI that does it?